### PR TITLE
[WIP] Make searching tags case insensitive

### PR DESCRIPTION
**Pull request by thearifismail**
This PR implements https://issues.redhat.com/browse/ESSNTL-570

### CODE REVIEW

**Comment by thearifismail** [comment]
@fijshion can you please take a look at it and see if I am on the right path?  Thanks.

File: src/schema/schema.graphql [hunk 1]
```diff
@@ -309,9 +309,9 @@ type Hosts {
 
 "Structured representation of a tag"
 type StructuredTag {
-    namespace: String,
-    key: String!,
-    value: String
+    namespace: FilterStringWithWildcardWithLowercase,
```
Comment (fijshion) [inline src/schema/schema.graphql:312]: I don't think these changes aren't needed. When I run the code with this change I get this error: `The type of StructuredTag.value must be Output Type but got: FilterStringWithWildcardWithLowercase.`

**Comment by fijshion** [comment]
I'm not super familiar with this code base so I'm learning with you. After looking at the JIRA, I think what needs to be updated is [this resolver](https://github.com/RedHatInsights/xjoin-search/blob/47a42a2744198d5c472194404b9fab24f0ecbc79/src/generated/graphql.ts#L395). That looks to be what handles filtering a list of tags. So I think a new function similar to [this one](https://github.com/RedHatInsights/xjoin-search/blob/47a42a2744198d5c472194404b9fab24f0ecbc79/src/resolvers/inputString.ts#L42) but supporting wildcards will need to be created.

File: src/schema/schema.graphql (hunk 1 above)
Comment (thearifismail) [reply src/schema/schema.graphql:312]: yes.  For `type StructuredTag`, I had to put back `String` as the type for namespace, key, and value; leaving the rest as in the new form.   Now the graphql server starts.  Basically, I need to learn GraphQL, apollo, resoolvers, and generator, inputs and outputs

Thanks for the tip.

**Pull request closed without merging**